### ISSUE TITLE
[Suggestion] Mirror docker image on dockerhub (for IPv6 download support)

### ISSUE THREAD

**Issue by lerokko** [mliezun/caddy-snake#46]
I just learned the hard way after setting up my vps for 20 minutes that **ghcr.io still does not support ipv6**. I have read people worked around this by also providing docker images via dockerhub as it supports IPv6.

I think it would be nice to have this so systems without ipv4 can still download the image.

**Comment by mliezun** [comment]
Nice suggestion!

I'll add docker hub support.

**Comment by mliezun** [comment]
Done ✅ 

Now image is available in docker hub: [https://hub.docker.com/r/mliezun/caddy-snake](https://hub.docker.com/r/mliezun/caddy-snake)

Also updated readme so that users are aware where they can get the image from. Both ghcr and docker hub.

Can you try it out and see if this resolves the problem that you were having?

**Comment by lerokko** [comment]
Yes, with only ipv6 access `docker pull` does work with the image from docker hub. Thanks for the quick resolve!
